app/assets/stylesheets/simple-sidebar.scss

```diff
@@ -3,90 +3,129 @@
  * Copyright 2013-2016 Start Bootstrap
  * Licensed under MIT (https://github.com/BlackrockDigital/startbootstrap/blob/gh-pages/LICENSE)
  */
-/* Toggle Styles */
-/* Sidebar Styles */
 
 $sidebar-size: 200px;
+$bg-default:   #000;
+$bg-selected:  #222;
+$bg-hover:     #333;
+$fg-default:   #bbb;
+$fg-hover:     #fff;
```

Comment:
put in this file https://github.com/Codeminer42/cm42-central/blob/master/app/assets/stylesheets/_variables.scss :)